City-of-Helsinki/kukkuu-admin
# KK-595 | Add send button

## Description

In these changes I integrate the message sending API and provide an view layer that allows the user to send messages using the API.

## Context

This feature allows users to send messages using the admin UI.

[KK-597](https://helsinkisolutionoffice.atlassian.net/browse/KK-597)

## How Has This Been Tested?

This has been tested manually.

## Manual Testing Instructions for Reviewers

As a logged in admin user in message details view

1. Expect to see send button
1. Click on send button
1. Expect to see a confirmation modal
1. Confirm sending
1. Expect to be taken into list view
1. Expect sent message to be displayed as sent
1. Go back into messages details view by clicking it from the list
1. Expect send button to be missing

## Screenshots

<img width="1680" alt="Screenshot 2020-10-26 at 10 25 40" src="https://user-images.githubusercontent.com/9090689/97149634-b3cb7500-1775-11eb-8881-5474768f2f6c.png">
<img width="1680" alt="Screenshot 2020-10-26 at 10 25 48" src="https://user-images.githubusercontent.com/9090689/97149641-b6c66580-1775-11eb-8279-215e42b45a12.png">
<img width="1680" alt="Screenshot 2020-10-26 at 10 26 01" src="https://user-images.githubusercontent.com/9090689/97149643-b7f79280-1775-11eb-999e-015ae0b2a995.png">
